modules/theme/src/theme-core/definitions/modules/hint.less

```diff
@@ -29,12 +29,14 @@
 
 @uiHintColor: @doveGray;
 @uiHintFontSize: 12px;
-@uiHintMargin: 10px 0;;
+@uiHintMargin: 10px 0;
+@uiHintLineHeight: 1.5;
```

Comment:
Hope this won't cause style issues in other places Hint has been used :)

Reply:
Yes, in this style I have added the line height of the body to the hint.